webpack.config.js

```diff
@@ -41,13 +40,12 @@ const commonSettings = {
     // Alternately, we can run rm -rf dist/ as
     // part of the package.json scripts.
     new CleanBuild(['dist']),
-    new ExtractTextPlugin('styles.css'),
+    new MiniCssExtractPlugin({
+      filename: 'styles.css',
+    }),
     new webpack.DefinePlugin({
       loadA11y: process.env.loadA11y || false,
       appEnv: JSON.stringify(appEnv),
-      'process.env': {
```

Comment:
What happened to this variable?

Reply:
Actually, ignore that change. I am going to revert it.